sessions/forecasting-concepts.qmd

```diff
@@ -73,11 +72,10 @@ This introduces new and different opportunities and complexities for modeling.
 # Getting acquainted with ILI
 
 Influenza-like illness (ILI) is a commonly used epidemiological surveillance measure for tracking the influenza season.
-An individual is classified as having ILI if they have a fever AND an additional "flu-like" symptom, such as a cough, headache, sore throat, etc...
-In the US, ILI data has been collected in a more or less consistent way for over 20 years at the national and state level.
+An individual is classified as having ILI if they have a fever AND an additional "flu-like" symptom, such as a cough, headache, sore throat, etc... In the US, ILI data has been collected in a more or less consistent way for over 20 years at the national and state level.
```

Comment:
I had separated these sentences upon a suggestion from Sam, A as I think it's easier to see the edits to sentences if they are part of separate lines in the .qmd file. in this one instance I think it's fine to combine them onto one line, but just mention this as a general guideline for style that we are trying to follow.